engine/beef_tx_bytes.go

```diff
@@ -62,6 +62,9 @@ func toBeefBytes(tx *trx.Transaction, bumps BUMPs) []byte {
 	bumpIdx := getBumpPathIndex(tx, bumps)
 	if bumpIdx > -1 {
 		txBeefBytes = append(txBeefBytes, hasBUMP)
+		if bumpIdx < 0 {
+			return nil
+		}
```

Comment:
what integer value could be  greater then -1 but lower then 0?

Reply:
I would recommend to supress this error here. 
Because I don't see any reason for that `if`.